public/app/features/backup/components/StorageLocations/RemoveStorageLocationModal/RemoveStorageLocationModal.types.ts

```diff
@@ -0,0 +1,9 @@
+import { StorageLocation } from '../StorageLocations.types';
+
+export interface RemoveStorageLocationModalProps {
+  location?: StorageLocation;
```

Comment:
Is it possible to delete a `StorageLocation` without an actual `StorageLocation`?

Reply:
Yep, it isn't. Anyway I had to change it to `StorageLocation | null`, because of the `useState(null)`